package.json

```diff
@@ -62,6 +62,7 @@
     "winston": "^3.2.1"
   },
   "dependencies": {
+    "@azure/storage-blob": "^12.2.0-preview.1",
```

Comment:
Do you want a stable or preview release for this dependency?

Reply:
We need this version in order to use blob versioning later on. At least that's what I'm seeing. I'll make a note to revisit this once v12.2 goes live.